config/routes.rb

```diff
@@ -10,5 +10,8 @@
     post "/login", to: "sessions#create"
     delete "/logout", to: "sessions#destroy"
     resources :users
+    resources :account_activations, only: [:edit]
+    resources :password_resets, only: [:new, :create, :edit, :update]
```

Comment:
đổi sang 
```suggestion
    resources :password_resets, only: %i(new create edit update)
```
tương tự những chỗ khác